webpack.config.js

```diff
@@ -1,23 +1,22 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
```

Comment:
- [ ] OPTIONAL
_Mode_ option is not set for _webpack_. Kindly set the `mode` and either assign its value to `development` or `production` within the _module.exports_

**Webpack config :**
![image](https://user-images.githubusercontent.com/12745474/154247540-9ff109e9-e166-47df-9602-1edcc48d92e5.png)

Reply:
fixed.